data/mods/Xedra_Evolved/mutations/werewolf_traits.json

```diff
@@ -0,0 +1,203 @@
+[
+  {
+    "//": "Werewolf traits should focus on hunting, fighting extradimensional threats, the moon, battle fury, shapeshifting",
+    "type": "mutation",
+    "id": "WEREWOLF_MANA_TRAIT",
+    "name": { "str": "Werewolf Mana", "//~": "NO_I18N" },
+    "points": 0,
+    "description": { "str": "Your mana is calculated differently because you're a werewolf.", "//~": "NO_I18N" },
+    "starting_trait": false,
+    "purifiable": false,
+    "valid": false,
+    "player_display": false,
+    "enchantments": [
+      {
+        "values": [ { "value": "MAX_MANA", "add": { "math": [ "(-1000 * (1 + ( (0.2 + (u_val('intelligence') * 0.1 ) - 1) ) )" ] } } ]
+      }
+    ]
+  },
+  {
+    "//": "Tier 1 traits begin here",
+    "type": "mutation",
+    "id": "WEREWOLF_SMELL_NEARBY_ENEMIES",
+    "name": { "str": "Scent the Prey" },
+    "points": 2,
+    "description": "Your sense of smell is heightened even in human form, allowing you to smell nearby enemies. In war or wolf form, this range is extended.",
+    "starting_trait": false,
+    "purifiable": false,
+    "valid": false,
+    "enchantments": [
+      {
+        "condition": { "or": [ "npc_is_npc", { "and": [ { "npc_has_species": "FERAL" }, { "not": { "npc_has_species": "ZOMBIE" } } ] } ] },
```

Comment:
This one is still a condition of enchantment, like ACTIVE or ALWAYS
You need to move special_vision conditions under the `special_vision` block (enchantments can evaluate only you as `u`, and can't use `npc`)

Reply:
Looks like you caught me right as I was fixing that